pubsub/driver/driver.go

```diff
@@ -139,15 +136,15 @@ type Subscription interface {
 	// err will always be a non-nil error returned from ReceiveBatch or SendAcks.
 	IsRetryable(err error) bool
 
-	// As allows providers to expose provider-specific types.
-	//
-	// See
-	// https://github.com/google/go-cloud/blob/master/internal/docs/design.md#as
-	// for more background.
+	// As converts i to provider-specific types.
+	// See https://godoc.org/gocloud.dev#As for background information, the "As"
```

Comment:
In `driver` packages, the user is the implementer of the interface, not the user of it, so the example code isn't as useful. How about something like:

```
// As converts i to provider-specific types.
// See https://godoc.org/gocloud.dev#As for background information, and
// existing driver implementations for implementation examples.
// Please also include the types that the driver supports for As functions
// in the package-level docstring; again, see existing drivers for examples.
```

Reply:
OK, I'll just shorten it to point to doc.go, since the other tips are already given there. I feel the new proposal is too lengthy for this